app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :post, class_name: 'User'
+
+  def update_post_like_counter
+    post.increment(:likescounter)
+  end
```

Comment:
- [ ] [OPTIONAL] Please consider using one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_like_counter` method every time a new post is created and saved to the database, then make the update_post_like_counter method private